tests/containers/host_configuration.pm

```diff
@@ -30,7 +30,7 @@ sub run {
         disable_and_stop_service(opensusebasetest::firewall, ignore_failure => 1);
     }
     else {
-        assert_script_run "dhclient -v";
+        assert_script_run "dhclient -v" unless get_var("HDD_1") =~ /rhel/;
```

Comment:
I'd add a comment here as well, so that we know in a month from know why we added this. Something like

    # Re-running dhclient on RHEL is confusing the routing tables

or something similar. Apart from that the PR looks good to me, I have troubles in understanding the VR. I'm assuming https://openqa.suse.de/tests/6645363 is the only relevant one? And the failure in `podman_image` there is not related to this PR.

Reply:
> I'd add a comment here as well, so that we know in a month from know why we added this. Something like
> 
> ```
> # Re-running dhclient on RHEL is confusing the routing tables
> ```

Ready for review

> 
> or something similar. Apart from that the PR looks good to me, I have troubles in understanding the VR. I'm assuming https://openqa.suse.de/tests/6645363 is the only relevant one? And the failure in `podman_image` there is not related to this PR.

correct. the podman problem is different and i think there is separate ticket as it fails elsewhere as well. 
Also tests/6645363 is the main VR but i used to try to run a few random ones usually to catch side effect/breaking changes

Reply:
Can we just just disable this module in rhel?

Reply:
first sorry i didnt see this earlier. 
this is also possible. i will provide a commit even if that one is already merged.